configs/platforms/ubuntu-16.10-amd64.rb

```diff
@@ -0,0 +1,12 @@
+platform "ubuntu-16.10-amd64" do |plat|
+  plat.servicedir "/lib/systemd/system"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "systemd"
+  plat.codename "yakkety"
+
+  plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/debian/pl-build-tools-release-#{plat.get_codename}.deb"
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive; apt-get update -qq; apt-get install -qy --no-install-recommends build-essential devscripts make quilt pkg-config debhelper rsync fakeroot"
+  plat.install_build_dependencies_with "DEBIAN_FRONTEND=noninteractive; apt-get install -qy --no-install-recommends "
+  plat.vmpooler_template "ubuntu-1610-x86_64"
+  plat.output_dir File.join("deb", plat.get_codename, "PC1")
```

Comment:
I'm not entirely sure about this inconsistency either, but the ubuntu platforms seem to all add it, while the feodora ones to do. @McdonaldSeanp may have more backstory on that.